djangocms_content_expiry/compat.py

```diff
@@ -0,0 +1,6 @@
+import django
+
+from packaging.version import Version
+
+
+DJANGO_4_2 = Version(django.get_version()) >= Version('4.2')
```

Comment:
I recommend to follow CMS `cms/utils/compat/__init__.py`  to define the DJANGO_4_2.

`
DJANGO_3 = Version(DJANGO_VERSION) < Version('4.0')
DJANGO_4_1 = Version(DJANGO_VERSION) < Version('4.2')
DJANGO_4_2 = Version(DJANGO_VERSION) < Version('4.3')
`

use `if not DJANGO_4_1 and DJANGO_4_2` to limit the code to work only for django 4.2

Reply:
Updated